update-extension-dependencies.sh

```diff
@@ -16,14 +16,19 @@ then
   echo ''
   mvn -q -e clean package -f "${PRG_PATH}/devtools/bom-descriptor-json" -Denforcer.skip $*
 else
-  read -n1 -p 'Did you build the entire project? [y/n] ' ANSWER
+  read -n1 -p 'Build the entire project with relocations? [y/n] ' ANSWER
   echo ''
-  if [ "${ANSWER}" != y ]
+  if [ "${ANSWER}" == y ]
   then
     echo ''
-    echo 'Building entire project...'
+    echo 'Building entire project with relocations...'
     echo ''
-    mvn -q -e -Dquickly -T0.8C -f "${PRG_PATH}" $*
+    mvn -q -e -Dquickly -Dno-test-modules -Prelocations -T0.8C -f "${PRG_PATH}" $*
+  else
+    echo ''
+    echo 'Aborted!'
```

Comment:
Btw, the idea behind kinda forcing the build is that I don't expect anyone to have built with `-Prelocations` before calling this script.